rpctest/memwallet.go

```diff
@@ -551,8 +551,8 @@ func (m *memWallet) ConfirmedBalance() dcrutil.Amount {
 }
 
 // keyToAddr maps the passed private to corresponding p2pkh address.
-func keyToAddr(key chainec.PrivateKey, net *chaincfg.Params) (dcrutil.Address, error) {
-	pubKey := chainec.Secp256k1.NewPublicKey(key.Public())
+func keyToAddr(key *secp256k1.PrivateKey, net *chaincfg.Params) (dcrutil.Address, error) {
+	pubKey := secp256k1.NewPublicKey(key.Public())
```

Comment:
No need to pull out the X and Y of the public key just to recreate it and cause more allocations that have to be GC'd.  Just cast it.

```Go
pubKey := (*secp256k1.PublicKey)(&key.PublicKey)
```